Uint_testingDIO/test.c

```diff
@@ -0,0 +1,155 @@
+/*
+ * test.c
+ *
+ *  Created on: Feb 20, 2020
+ *      Author: hazem ebrahim
+ */
+
+
+#include "DIO_TS/DIO.h"
+#include <stdio.h>
+
+DIO_Cfg_s cfg_test;
+
+void DIO_initTesting(uint8_t gpio, uint8_t bit, uint8_t dir, uint8_t expected_output)
+{
+	static int initTestCounter = 1;
+	uint8_t output;
```

Comment:
Name your variables according to the guidelines, make it u8_output instead of output.